spring-boot/admin/src/main/resources/templates/admin/fragments/proxies/list.html

```diff
@@ -142,6 +143,7 @@ <h2 th:text="#{admin.proxies.list.title}">[Proxies Administration]</h2>
                                     <td class="text-center "><a th:href="|@{/admin/proxies/}${proxy.id}|"
                                                                 class="fa fa-bar-chart"/>
                                     </td>
+                                    <td th:text="${proxy.timeOfLastSuccessfulCallToAdmin != null ? #dates.format(new java.util.Date(proxy?.timeOfLastSuccessfulCallToAdmin)) : 'No sign of interaction'}"></td>
```

Comment:
Maybe move 'No sign of interaction' into messages.properties?

Reply:
Done, however I can't test if it works, as I am putting a message #{} inside of a Elvis operator condition